frontend/src/locales/fr.json

```diff
@@ -0,0 +1,183 @@
+{
+	"Create": "Créer",
+	"Matches": "Matchs",
+	"Game Servers": "Serveurs de jeu",
+	"Login": "Connexion",
+	"Logout": "Déconnexion",
+	"Chat": "Tchat",
+	"Send chat message...": "Envoyer un message dans le tchat...",
+	"Select Preset": "Sélectionner Préréglage",
+	"Delete preset": "Supprimer préréglage",
+	"Preset Name": "Nom Préréglage",
+	"Can be used by": "Utilisable par",
+	"Everybody": "Tout le monde",
+	"Only logged in users": "Utilisateurs connectés seulement",
+	"Add new preset": "Ajouter nouveau préréglage",
+	"Update preset": "Enregistrer préréglage",
+	"The map pool has too few maps.": "La liste de cartes est trop courte.",
+	"Teams": "Équipes",
+	"Team A Name": "Nom Équipe A",
+	"Team B Name": "Nom Équipe B",
+	"Game Server": "Serveur de jeu",
+	"Use Own Game Server": "Utiliser son propre serveur de jeu",
+	"Game Server IP": "IP Serveur",
+	"Game Server Port": "Port Serveur",
+	"Game Server RCON Password": "Mot de passe RCON Serveur",
+	"Map Pool": "Liste Cartes",
+	"Election Steps": "Étapes Élection",
+	"Quick Load:": "Chargement Rapide :",
+	"Best of 1": "Best of 1",
+	"Best of 3": "Best of 3",
+	"Empty": "Vide",
+	"Add new step": "Ajouter un étape",
+	"RCON Commands": "Commandes RCON",
+	"Init": "Initialisation",
+	"Executed only once: when the match is created": "Exécuté une seule fois : lors de la création du match",
+	"Knife": "Knife",
+	"Executed at the start of a knife round": "Exécuté au début d'un knife round",
+	"Match": "Match",
+	"Executed at the start of each match map": "Exécuté au début de chaque carte du match",
+	"End": "Fin",
+	"Executed only once: after the end of the last map": "Exécuté une seule fois : à la fin de la dernière carte",
+	"Advanced Settings": "Paramètres Avancés",
+	"Hide": "Cacher",
+	"Show": "Montrer",
+	"Mode": "Mode",
+	"Single match (stops when match is finished)": "Match unique (s'arrête quand le match est fini)",
+	"Loop mode (starts again after match is finished)": "Mode boucle (recommence après la fin du match)",
+	"Match End Action": "Action Fin Match",
+	"None": "Aucune",
```

Comment:
Bro every word is gendered in French

Reply:
What is the context of this "None" ?

Reply:
It's in the dropdown menu for "Match End Action" right above.
If in the future more text with just "None" are added, but they have a different translation, we can define them with an identifier in the code instead of the english text directly, and define their english meaning in the `en.json` file. So this is fine.